cmd/root.go

```diff
@@ -85,13 +90,19 @@ func NewRootCmd(ctx context.Context) *cobra.Command {
 		Short:         "Calyptia Cloud CLI",
 		SilenceErrors: true,
 		SilenceUsage:  true,
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			if !vercheck {
+				versionCheck(cmd)
+			}
+		},
 	}
 
 	cmd.SetOut(os.Stdout)
 
 	fs := cmd.PersistentFlags()
 	fs.StringVar(&cloudURLStr, "cloud-url", cfg.Env("CALYPTIA_CLOUD_URL", cloudURLStr), "Calyptia Cloud URL")
 	fs.StringVar(&token, "token", cfg.Env("CALYPTIA_CLOUD_TOKEN", token), "Calyptia Cloud Project token")
+	fs.BoolVar(&vercheck, "disable-version-check", false, "disable version check ")
```

Comment:
Let's add an env var as well so we don't have to keep adding the flag or set up an alias.

Reply:
done

Reply:
+1 to this

Reply:
https://github.com/calyptia/cli/pull/714/files#diff-ab967ab1a2f3a1b769106eeb7bfe892ef0e81d1d27811fa15be08e6749feee1fR31